src/source/PeerConnection/PeerConnection.c

```diff
@@ -1521,6 +1522,19 @@ STATUS setLocalDescription(PRtcPeerConnection pPeerConnection, PRtcSessionDescri
     return retStatus;
 }
 
+STATUS createRollingBufferConfig(PRtcRtpTransceiver pRtcRtpTransceiver, PRtcMediaStreamTrack pRtcMediaStreamTrack, DOUBLE rollingBufferDurationSec,
```

Comment:
Thinking this method should be called something more along the lines of "configureTransceiver" and we can pass in whatever settings. It makes more sense given the CHK_WARN below.